projects/aws/eks-anywhere/eks-anywhere-diagnostic-collector/Makefile

```diff
@@ -0,0 +1,62 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+BUILD_LIB=$(BASE_DIRECTORY)/build/lib
+GIT_TAG=$(shell cat GIT_TAG)
+
+GOPROXY_DNS?=https://proxy.golang.org
```

Comment:
We can remove the Goproxy override since we don't do Go builds for the collector